.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: release
+
+on:
+  push:
+    branches: [master]
```

Comment:
I just noticed in a final check that this line isn't needed at the moment, I probably introduced it while thinking about publishing to test.pypi.org from the `master` branch.

```suggestion
```